plugin/src/main/java/com/denizenscript/denizen/objects/properties/item/ItemProperty.java

```diff
@@ -33,4 +35,10 @@ public void setItemStack(ItemStack item) {
     public void setItemMeta(ItemMeta meta) {
         object.setItemMeta(meta);
     }
+
+    public <T extends ItemMeta> void editMeta(Class<T> metaType, Consumer<T> editor) {
+        T meta = metaType.cast(getItemMeta());

```

Comment:
Is there a specific reason to use this over just `(T)`? I ask specifically because I generally worry about reflection calls having unexpected performance issues, so it's preferable to avoid if not needed

Reply:
Changed to a direct cast